main.go

```diff
@@ -343,6 +367,15 @@ func transformCompile(args []string) ([]string, error) {
 
 	blacklist := buildBlacklist(files, info, pkg)
 
+	pkgDebugDir := ""
+	if envGarbleDebugDir != "" {
+		pkgDebugDir = filepath.Join(envGarbleDebugDir, pkgPath)
+		err = os.MkdirAll(pkgDebugDir, 0755)
```

Comment:
we already do the mkdir in the first execution, so remove this. this is also racy if we're garbling many packages in parallel, which is common.

Reply:
In the first execution we create the debugdir e.g. `./test`, in this line the debugdir for the package is created which is debugdir + importpath e.g. `./test/github.com/mvdan/project` or `./test/github.com/mvdan/project/package/subpackage`.

Reply:
Ah, gotcha. Hm. It could still be racy, though; imagine if two parallel processes try to MkdirAll the paths `foo/bar` and `foo/baz`. They would race to create the directory `foo`.

I'm not sure how to best deal with that, though. I guess we can deal with that if anyone encounters an error.

Reply:
@mvdan from the Documentation "If path is already a directory, MkdirAll does nothing and returns nil."